packages/application/style/buttons.css

```diff
@@ -94,6 +94,7 @@ button.jp-mod-styled.jp-mod-warn:focus-visible {
 
 .jp-Button-flat:hover {
   background-color: var(--jp-warn-color-hover, rgba(153, 153, 153, 0.1));
+  color: rgb(255, 255, 255) !important;
```

Comment:
Is it possible to do this using a color variable, such as in the previous line, and without using the `!important` string?